src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -481,5 +489,80 @@ private static string GetTempAssemblyName(AssemblyName parent, string ns)
         {
             return parent.Name + ".XmlSerializers" + (ns == null || ns.Length == 0 ? "" : "." + ns.GetHashCode());
         }
+
+        private static void ParseReferences(string value, Dictionary<string, string> dictionary)
+        {
+            var referencelist = new List<string>();
+            if (value.Length > 0)
+            {
+                string[] entries = value.Split(new char[] { ';' });
```

Comment:
>Split(new char[] { ';' } [](start = 41, length = 24)

string.Split has an overload which takes a single char. You are using it on line 529.